### ISSUE TITLE
Clickable Tags

### ISSUE THREAD

**Issue by tyeeman** [verless/verless#232]
Just trying this now. I see tags on your own site but they are not clickable. That's the only thing missing for me. I recently made a ssg myself using the worst language on the planet, a Windows batch file, including tags, but unfortunately Netlify does not support batch files, but they do support go exe's and verless would work great on Netlify. Could you add clickable tags feature?

**Comment by tyeeman** [comment]
Update - Just gone through your tutorial and found a couple of glitches - 

1. When creating the "light" theme, the "assets" folder is not created.
2. When creating "content" article, creating it from the main verless folder gives error because it's looking for "content/....". This means I have to go into the "content" folder first then run the command, which works. If I added the project name in front of the markdown filename I don't think it would work. I haven't tried that yet. If I added the full path to the project folder that might work. I will try but I would like to do it the way you intended.

**Comment by dominikbraun** [comment]
> Could you add clickable tags feature?

Well, the tags on my website aren't clickable because this wasn't possible at that time, but in fact, you can create a link to a tags page like so:

```html
{{range $tag := .Page.Tags}}
    <a href="/tags/{{$tag}}">{{$tag}}</tag>
{{end}}
```

We may introduce a `$tag.Href` field in the future to simplify this.

**Comment by dominikbraun** [comment]
> When creating the "light" theme, the "assets" folder is not created.

This just has been fixed with [verless v0.5.1](https://github.com/verless/verless/releases/tag/v0.5.1).

**Comment by aligator** [comment]
I noticed that the example does not work like that because of tags with uppercase.

This get's fixed with the PR which introduces a new property tag.Href:
```html
{{range $tag := .Page.Tags}}
    <a href={{$tag.Href}}>{{$tag}}</a>
{{end}}
```
